packages/calcite-components/src/utils/sortableComponent.ts

```diff
@@ -5,6 +5,8 @@ export interface DragDetail {
   toEl: HTMLElement;
```

Comment:
Sidebar: we should probably add a conventions entry about meaningful event details and when they should be allowed. This is a perfect example of how this info helps add context behind an event that would be hard to represent in another way.

Reply:
Yes a convention document on this would be super useful!